packages/core/src/App/Containers/ResetEmailModal/confirm-email-modal.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { SentEmailModal } from '@deriv/account';
+import { WS } from 'Services';
+import PropTypes from 'prop-types';
+
+export const ConfirmEmailModal = ({
+    onClose,
+    is_open,
+    prev_email,
+    changed_email,
+    verification_code,
+    setEmailError,
+    setErrorMessage,
+}) => {
+    const [email_request, setEmailRequest] = React.useState(null);
+    const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
+    const [is_modal_open, setIsModalOpen] = React.useState(is_open);
+
+    const handleSubmit = () => {
+        const api_request = {
+            change_email: 'verify',
+            new_email: changed_email,
+            verification_code,
+        };
+
+        setEmailRequest(prev => ({ ...prev, ...api_request }));
+
+        WS.changeEmail(api_request).then(response => {
+            if (response.error) {
+                setEmailError(true);
+                onClose();
+                setErrorMessage(response.error.message);
+            } else {
+                setIsSendEmailModalOpen(true);
+            }
+            setIsModalOpen(false);
+        });
+    };
+
+    const resendEmail = () => {
+        WS.changeEmail(email_request);
+    };
+
+    if (is_send_email_modal_open) {
+        return (
+            <SentEmailModal
+                is_open={is_send_email_modal_open}
+                onClose={() => setIsSendEmailModalOpen(false)}
+                identifier_title={'Change_Email'}
+                onClickSendEmail={resendEmail}
+                has_live_chat={true}
+                is_modal_when_mobile={true}
+            />
+        );
+    }
+    return (
+        <Modal
+            is_open={is_modal_open}
+            should_header_stick_body
+            title={<Localize i18n_default_text='Are you sure?' />}
+            toggleModal={onClose}
+            width='440px'
+        >
+            <React.Fragment>
+                <div className='email-confirmation'>
+                    <Text as='p' color='prominent' size='xs' align='left'>
+                        <Localize
+                            i18n_default_text='Are you sure you want to update email <0>{{prev_email}}</0> to <1>{{changed_email}}</1>?'
+                            values={{ prev_email, changed_email }}
+                            s
```

Comment:
what is this?

Reply:
i cant have the same name as new_email as its conflicting with the new_email from api request